src/main/java/org/sonarlint/intellij/config/global/wizard/OrganizationStep.java

```diff
@@ -38,16 +38,16 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.sonarlint.intellij.tasks.GetOrganizationTask;
-import org.sonarsource.sonarlint.core.serverapi.organization.ServerOrganization;
+import org.sonarsource.sonarlint.core.clientapi.backend.connection.org.OrganizationDto;
 
 import static javax.swing.JList.VERTICAL;
 
 public class OrganizationStep extends AbstractWizardStepEx {
   private final WizardModel model;
-  private JList<ServerOrganization> orgList;
+  private JList<OrganizationDto> orgList;
```

Comment:
I'm a bit irritated but also not too deep into that matter: Why does `OrganicationDto` not implement the interface `ServerOrganization`? We could have used it then anyway ^^

Reply:
I guess a DTO should not have such logic 🤷🏻